frappe/core/doctype/data_import/importer.py

```diff
@@ -449,8 +449,8 @@ def parse_next_row_for_import(self, data):
 			data_without_first_row = data[1:]
 			for row in data_without_first_row:
 				row_values = row.get_values(parent_column_indexes)
-				# if the row is blank, it's a child row doc
-				if all([v in INVALID_VALUES for v in row_values]):
+				# if the row is blank or same content as the previous parent row, it's a child row doc
+				if all([v in INVALID_VALUES for v in row_values]) or row_values == parent_row_values:
```

Comment:
This logic was purposefully removed because some people might assume each row will be imported separately. Keeping the cells blank is less confusing.